go/internal/openapi/api_endpoint.go

```diff
@@ -1156,12 +1156,14 @@ func (r ApiV1EndpointGetStatsRequest) Execute() (EndpointStats, *_nethttp.Respon
  * @param endpointId The ep's ID or UID
  * @return ApiV1EndpointGetStatsRequest
  */
-func (a *EndpointApiService) V1EndpointGetStats(ctx _context.Context, appId string, endpointId string) ApiV1EndpointGetStatsRequest {
+func (a *EndpointApiService) V1EndpointGetStats(ctx _context.Context, appId string, endpointId string, since *time.Time, until *time.Time) ApiV1EndpointGetStatsRequest {
```

Comment:
So I'm sure this is just my ignorance here, but we're ok w/ breaking changes to the "internal" files?

Reply:
It's been a while since I've used go, but iirc everything in `/internal` is only visible to the package, not users of the package. So folks using our libs can't call this function anyway.

And then the functions they do call (in `go/endpoint.go`) haven't had their interface change at all

Reply:
Yeah. The gist is, Go won't let you import any package under an `internal/` path segment if the import isn't coming from inside the module itself. Dependents can't access that stuff.